### PR TITLE
Add client tests

### PR DESCRIPTION
**Pull request by cmebarrow**
Adds spec tests for a HTTP nukleus acting as a client: client equivalents of all the server tests.

### CODE REVIEW

File: src/main/scripts/org/reaktivity/specification/nukleus/http/streams/rfc7230/message.format/response.with.headers/client/nukleus.rpt [hunk 1]
```diff
@@ -24,7 +24,7 @@ property targetInputEst ${nuklei.streams("http", "target")}
 property sourceInputEst ${nuklei.streams("source", "http#target")}
 
 property newTargetOutputId ${nuklei:newStreamId()}
-property newSourceInputEstId ${nuklei:newStreamId()}
+property sourceInputEstId ${nuklei:newStreamId()}
```
Comment (jfallows) [inline src/main/scripts/org/reaktivity/specification/nukleus/http/streams/rfc7230/message.format/response.with.headers/client/nukleus.rpt]: The general naming strategy is to use `newXXX` if we generate via property section, or just `xxx` if we capture from the script body.

This one seems not to follow the naming convention.

**Review by jfallows — CHANGES_REQUESTED** [review]

File: src/main/scripts/org/reaktivity/specification/nukleus/http/streams/rfc7230/message.format/response.with.headers/client/nukleus.rpt (hunk 1 above)
Comment (cmebarrow) [reply src/main/scripts/org/reaktivity/specification/nukleus/http/streams/rfc7230/message.format/response.with.headers/client/nukleus.rpt]: oh I see I had misunderstood it as being connected with the type of route (e.g. route output new).

Comment (cmebarrow) [reply src/main/scripts/org/reaktivity/specification/nukleus/http/streams/rfc7230/message.format/response.with.headers/client/nukleus.rpt]: fixed in all new scripts

**Comment by cmebarrow** [comment]
@jfallows Ready for re-review

**Review by jfallows — APPROVED** [review]

**Pull request merged**